### ISSUE TITLE
Code context when deploying only binary

### ISSUE THREAD

**Issue by grongor** [getsentry/raven-go#249]
Hey, maybe I'm doing something wrong or I'm missing some config or whatnot, but I'm not able to get source code context to appear in the Sentrt when running the compiled binary on another machine (typical use: build on CI, deploy to another server (bin only), run there). The paths are relative to where I built the binary and the source code is missing (I see only file/function names).

![image](https://user-images.githubusercontent.com/972493/55668949-3679a000-5871-11e9-927a-8c86fd5f127b.png)

**Comment by micahcoffman** [comment]
+1 on being able to resolve context somehow when binary is compiled in CI and run as binary elsewhere

**Comment by kamilogorek** [comment]
It'd still require source code to correctly map the frames to a given file. Go binaries doesn't provide enough information to do that, as they are compiled down to machine code. It'd be possible to somehow perform mapping to external files, but environment running the binary would still need access to the corresponding files.

**Comment by micahcoffman** [comment]
Absolutely, but it'd be better than nothing to be able to provide some source code if a user wants a mapping. Like the ability to add sourcemaps for JS.
